include/snowflake/client.h

```diff
@@ -443,9 +453,9 @@ typedef struct SF_STMT {
     char request_id[SF_UUID4_LEN];
     SF_ERROR_STRUCT error;
     SF_CONNECT *connection;
-    void *qrf;
+    QueryResultFormat_t qrf;
```

Comment:
is this change compatible for users? our public API is changing there - don't we need * anymore?

Reply:
it was void* and I don't think application would use it without knowing the actual data type it points to.
It was actually pointing to QueryResultFormat_t type which was dynamically allocated, having the enum data directly would make much more sense.

Reply:
And that't the reason why we cannot merge it even now - right?